### PR TITLE
@ashfurrow => Adds support for sending select events to Adjust - fixes #749

### LINKED ISSUE #749: Integrate Adjust and set up event tracking
Hi team,

In the last few weeks we have been considering different business intelligence platforms for mobile app marketing analytics. After doing solid research and talking to our Apple's App Store Marketing contact, Jesse proposed that we proceed with the integration of a service called [Adjust](https://www.adjust.com/). You can find login credentials in the Engineering vault in 1Password.

Adjust would let us track different marketing campaigns' performance in both conversion rate (from clicks on campaign links to iTunes store to app downloads) and user behavior in the app. This is the [iOS SDK of Adjust](https://github.com/adjust/ios_sdk) -- their readme is a detailed documentation of the integration process. We are also in contact with an account manager which we can contact shall any questions arise. 

So far we have linked Adjust to Artsy and are in the process of generating links for different campaigns that we already have running (eg m.artsy banner that went live on friday, links to iTunes on Artsy, etc.). Next on the line is the integration of Adjust's SDK into our project and the set up of event tracking.

Events that we would be interested in adding the Adjust tracking calls to:
- [x] first user install
- [x] created account
- [x] sent artwork inquiry
- [x] bid button

Platform team doesn't currently have the bandwidth to set up a server for us which would store the Adjust event callbacks and get them into our data warehouse. We will discuss the prioritization of this as soon as they finish their current reporting infrastructure project (getting Segment data into Redshift). In the meantime, we can be only set up the Adjust trackEvents.

Have not discussed final prioritization for this with @katarinabatina and @alloy yet, but I imagine it will be something to tackle this coming sprint prior to the partnership with Sotheby's.

Huge thanks to @jessekedy for spearheading this integration which will be crucial in informing our growth strategies going forward!

### PR DESCRIPTION
**Pull request by orta**

### CODE REVIEW

File: Artsy/App/ARAppDelegate+Analytics.m [hunk 1]
```diff
@@ -368,7 +368,7 @@ - (void)setupAnalytics
                         },
                         @{
                             ARAnalyticsEventName: ARAnalyticsAuctionBidTapped,
-                            ARAnalyticsSelectorName: NSStringFromSelector(@selector(bidCompelted:)),
+                            ARAnalyticsSelectorName: NSStringFromSelector(@selector(bidCompleted:)),
```
Comment (orta) [inline Artsy/App/ARAppDelegate+Analytics.m]: screen shot 2015-08-24 at 5 53 14 pm

**Comment by anipetrov** [comment]
@jessekedy would you look at that!

File: Artsy/App/ARAppDelegate+Analytics.m (hunk 1 above)
Comment (ashfurrow) [inline Artsy/App/ARAppDelegate+Analytics.m]: :grimacing:

**Comment by ashfurrow** [comment]
Looks good.

**Comment by alloy** [comment]
So what’s going on here? Is this a partial implementation, can it be merged, what gives?

**Comment by ashfurrow** [comment]
Think it just never got merged.

**Comment by alloy** [comment]
Okidoki :+1:

**Pull request merged**